client/src/shared/icons/ChevronDown.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
```

Comment:
- as per standard please change file name `ChevronDownIcon`

i noticed some existing icons does not have `postfix` (`Icon`). Could apply it as well :bow: